specificatie/gba-openapi.yaml

```diff
@@ -226,7 +226,7 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/develop/specificatie/common.yaml#/components/responses/default"
       tags:
         - BRP historie bevragen
-# Daar waar mogelijk wordt nu verwezen naar componenten die in HC BRP Personen zijn gedefinieerd. Dat is echter alleen mogelijk als betreffende componenten zelf niet ook een $ref bevatten. 
+# Daar waar mogelijk wordt nu verwezen naar componenten die in HC BRP Personen zijn gedefinieerd. Dat is echter alleen mogelijk als betreffende componenten zelf niet ook een $ref bevatten.
```

Comment:
Hier ook gewijzigde description overnemen? En de vraag van Frank: doen we dit nog?

Reply:
Op enkele plaatsen wordt er in de OAS nog verwezen naar de Haal-Centraal-BRP-bevragen repository. Dat gaat om de GBAdatum en enkele foutmeldingen. Die zitten echter ook in de lokale common van BRP-historie-bevragen. 

Zal ik dat aanpassen zodat er geen enkele verwijzing naar een andere repository meer is ? Dat is volgens mij wat we willen. Ook het hier genoemde commentaar zal ik dan verwijderen. 

Wat mij betreft kan dat in deze PR mee, maar ik wil er ook een nieuwe PR voor maken indien gewenst.

Reply:
Net even een belrondje gedaan. Deze opmerking wordt verwijderd en de verwijzingen naar de BRP-bevragen repository (die nog deel in historie-bevragen zitten) worden vervangen door een verwijzing naar de common in historie-bevragen.

Hier maak ik een nieuwe PR voor aan.